nixos/modules/programs/ryzen-monitor-ng.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs, config, lib, ... }:
+
+with lib;
```

Comment:
this use of with should also be removed.

Reply:
Done